pom.xml

```diff
@@ -56,10 +56,6 @@
         </contributor>
     </contributors>
 
-    <prerequisites>
```

Comment:
Your change is objectively better, but it introduces additional differences between the upstream JUnit 4 (which has this `<prerequisites>` block) and our version.  I would like to keep diffs to a minimum.

Reply:
While the <prerequisites> block does work, it gets a warning which I was trying to eliminate:
[WARNING] The project junit:junit:jar:4.13.2-Daikon uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects.

It's fine to leave it in if you wish.